src/java.desktop/share/classes/java/awt/font/FontRenderContext.java

```diff
@@ -290,51 +291,29 @@ public Object getFractionalMetricsHint() {
     /**
      * Return true if obj is an instance of FontRenderContext and has the same
      * transform, antialiasing, and fractional metrics values as this.
-     * @param obj the object to test for equality
+     * @param o the object to test for equality
      * @return {@code true} if the specified object is equal to
      *         this {@code FontRenderContext}; {@code false}
      *         otherwise.
      */
-    public boolean equals(Object obj) {
-        try {
-            return equals((FontRenderContext)obj);
-        }
-        catch (ClassCastException e) {
-            return false;
-        }
-    }
-
-    /**
-     * Return true if rhs has the same transform, antialiasing,
-     * and fractional metrics values as this.
-     * @param rhs the {@code FontRenderContext} to test for equality
-     * @return {@code true} if {@code rhs} is equal to
-     *         this {@code FontRenderContext}; {@code false}
-     *         otherwise.
-     * @since 1.4
-     */
-    public boolean equals(FontRenderContext rhs) {
```

Comment:
The public methods in the public classes in the `java/awt/*` package are part of the public API, it is not possible to delete such methods.

Reply:
Whoops. Thanks for catching that.